internal/server/v0/internalplace/golden/get_place_page_data/asm.sample.json

```diff
@@ -1727,1056 +1481,6 @@
             "observationPeriod": "P1Y"
           }
         },
-        "CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased": {
```

Comment:
Is this expected?

Reply:
For the (population/covid) deletions in the place page, I think that is because frequent has been rebuilt without the non-sharded place-page cache (cl/488794608).

My guess is that the website code has switched to requesting a specific category in API, but these tests are still doing category-less calls.  So we should probably make "category" a required parameter and upgrade the tests.

For instance autopush indeed has covid deaths for https://autopush.datacommons.org/place/country/ASM

Reply:
Good point. I sent out https://github.com/datacommonsorg/mixer/pull/986 to fix that and can rebase this on top of that.

Reply:
The mixer fix is in now. The v0 APIs are removed there.